src/azure-cli/azure/cli/command_modules/acs/commands.py

```diff
@@ -24,14 +24,16 @@
 def load_command_table(self, _):
 
     container_services_sdk = CliCommandType(
-        operations_tmpl='azure.mgmt.containerservice.v2017_07_01.operations.'
```

Comment:
@arrownj This PR basically aims to add AKS(CONTAINERSERVICE) CLI into AzureStack profile using a specific API version. Previous CONTAINERSERVICE python SDK version is hardcoded here. I followed the same pattern as other services, to remove the version and add operation_group allowing multi-api support. However, the test didn't pass, and throw error: {'load_commands': ModuleNotFoundError("No module named 'azure.mgmt.containerservice.operations'")}

I am not sure how to fix it, can you please help to provide some guidelines?